lib/utils/wechat-mp.js

```diff
@@ -178,24 +312,28 @@ const fetchArticle = async (ctx, url, bypassHostCheck = false) => {
     url = normalizeUrl(url, bypassHostCheck);
     return await ctx.cache.tryGet(url, async () => {
         const response = await got(url);
-        const $ = cheerio.load(response.data);
+        const rootHtml = response.data;
+        const $ = cheerio.load(rootHtml);
 
         const title = ($('meta[property="og:title"]').attr('content') || '').replace(/\\r/g, '').replace(/\\n/g, ' ');
         const author = $('meta[name=author]').attr('content');
         let summary = $('meta[name=description]').attr('content');
         summary = summary !== title ? summary : '';
-        let description = fixArticleContent($('#js_content'));
+        const headImage = getAccountHeadImage(rootHtml);
+        const voiceInfo = detectVoiceInfo(rootHtml);
+        let description = await fixArticleContent($('#js_content'), false, voiceInfo);
 
         // No article get or article is too short, try the original url
         const originalUrl = detectOriginalArticleUrl($);
         if (originalUrl) {
             // try to fetch the description from the original article
             const originalResponse = await got(normalizeUrl(originalUrl, bypassHostCheck));
             const original$ = cheerio.load(originalResponse.data);
-            description += fixArticleContent(original$('#js_content'));
+            const originalVoiceInfo = detectVoiceInfo(originalResponse.data);
```

Comment:
Please do not aggressively match the whole HTML, but firstly using cheerio to select that part may contain these resources before performing matching.